### PR TITLE
V0.1 skirt mod for SSD1306 OLED

### PR DESCRIPTION
**Pull request by mpfl**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

A V0.1 skirt modified to accept a 0.96" SSD1306 OLED screen. This mod is heavily inspired by madros V0 OLED skirt, but adds the extra height required for the V0.1.

#### How was it tested? 

Printed in eSUN ABS+ following Voron printing standards and installed on V0.383. PR includes a photo of the installed mod.

#### Any background context you want to provide?

Nil

#### Screenshots (if appropriate)

Nil

#### Further notes

Nil

### CODE REVIEW

**Comment by mpfl** [comment]
Sorry, just realised that this would count as a "duplicate" because it's an update of an existing mod. Is that correct?

**Comment by jurisv** [comment]
> Sorry, just realised that this would count as a "duplicate" because it's an update of an existing mod. Is that correct?

V0.0 is obsolete, so this is for V0.1. Unless Original user of that mod updates it, it should not be counted as duplicate IMHO. Just my 2c.

File: printer_mods/mpfl/V0.1_skirt_OLED/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# V0.1 0.96" OLED skirt
+
+![Crappy photo](photo.jpg)
```
Comment (KiloQubit) [inline printer_mods/mpfl/V0.1_skirt_OLED/README.md:3]: Broken image, it's called png in your mod:

    Crappy photo

File: printer_mods/mpfl/V0.1_skirt_OLED/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+Inspired by [`madros/V0_skirt_OLED`](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/madros/V0_skirt_OLED), but for the V0.1. Please see that directory for installation instructions - no need to repeat them here.
+
+OpenSCAD file provided if you want to customise (perhaps you'd like to replace that Voron logo with your serial).
+You will need to grab [`Cowling_LOGO_ONLY_x1.STL`](https://github.com/VoronDesign/Voron-0/blob/V0.1/STLs/Toolheads/Mini_Afterburner/Cowling_LOGO_ONLY_x1.STL) and [`Skirt_x3.STL`](https://github.com/VoronDesign/Voron-0/blob/V0.1/STLs/Skirt_x3.STL) from [`VoronDesign/Voron-0`](https://github.com/VoronDesign/Voron-0/releases/tag/V0.1).
+
+v0.1-skirt-oled.scad is licensed with a GNU General Public License v3.0. All other file used for this mod are copyright their respective creators and are licensed separately.
```
Comment (KiloQubit) [inline printer_mods/mpfl/V0.1_skirt_OLED/README.md:10]: Please remove this paragraph, this whole repository is licensed GPL 3, which applies to all mods in here.

https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  Minor README fixes below.

File: printer_mods/mpfl/V0.1_skirt_OLED/README.md (hunk 1 above)
Comment (mpfl) [reply printer_mods/mpfl/V0.1_skirt_OLED/README.md]: Fixed in ffe868a

File: printer_mods/mpfl/V0.1_skirt_OLED/README.md (hunk 2 above)
Comment (mpfl) [reply printer_mods/mpfl/V0.1_skirt_OLED/README.md]: Fixed in ffe868a

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**